fedcloud_vm_monitoring/site_monitor.py

```diff
@@ -124,19 +126,42 @@ def get_user_email(self, egi_user):
             return f"{egi_user} not found in LDAP, has VO membership expired?"
         return self.user_emails[egi_user]
 
+    def get_public_ip(self, ip_addresses):
+        result = ""
+        for ip in ip_addresses:
+            if not ip.startswith("192.168") and not ip.startswith("172.16"):
```

Comment:
I suggest to use https://docs.python.org/3/library/ipaddress.html

Reply:
https://docs.python.org/3/library/ipaddress.html#ipaddress.IPv4Address.is_private

Reply:
Great suggestion, many thanks!